cassiopeia/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -14,7 +14,7 @@
 	http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task-3.0.xsd">
 
 	<mvc:annotation-driven />
-	<context:component-scan base-package="edu.asu.diging.gilesecosystem.cassiopeia,edu.asu.diging.gilesecosystem.util,edu.asu.diging.gilesecosystem.requests" />
+	<context:component-scan base-package="edu.asu.diging.gilesecosystem.cassiopeia,edu.asu.diging.gilesecosystem.util,edu.asu.diging.gilesecosystem.requests,edu.asu.diging.gilesecosystem.kafka.util,edu.asu.diging.gilesecosystem.util.properties" />
```

Comment:
it's the same here (as in Nepomuk). I guess I didn't do it the way it should be in Cepheus either, but that's no excuse ;)